osu.Game/Overlays/BeatmapListing/BeatmapSearchMultipleSelectionFilterRow.cs

```diff
@@ -160,13 +166,26 @@ protected override void UpdateState()
             {
                 Color4 colour = Active.Value ? ColourActive : ColourNormal;
 
-                if (IsHovered)
+                if (!Enabled.Value)
+                    colour = colour.Darken(1f);
+                else if (IsHovered)
                     colour = Active.Value ? colour.Darken(0.2f) : colour.Lighten(0.2f);
 
                 if (Active.Value)
                 {
-                    // This just allows enough spacing for adjacent tab items to show the "x".
-                    Padding = new MarginPadding { Left = 12 };
+                    if (Enabled.Value)
```

Comment:
Remove this IMO. Just disable the button, no need to hide things.

Reply:
Made less sense to have the 'x' button if it's disabled, but fine.

Reply:
https://github.com/ppy/osu/pull/31238/commits/050bf9ec6033b26a4a0cb6878738dc66346ba0b7f